lib/testutil/testutil.go

```diff
@@ -59,3 +64,139 @@ type NoopCloser struct{}
 func (NoopCloser) Close() error {
 	return nil
 }
+
+func IfNotCI[T any](notCiValue T, ciValue T) T {
+	if os.Getenv("CI") == "true" {
+		return ciValue
+	} else {
+		return notCiValue
+	}
+}
+
+func ConcatSlices[T any](slices ...[]T) []T {
+	// TODO when go >= 1.22.0: Replace with slices.Concat
```

Comment:
@emlun Our min go version is 1.22 now.

Reply:
Fixed in b470ce37cf462c4d5ba82e0d1b07706aea1ed463, thanks!